apps/src/netsim/NetSimSlider.js

```diff
@@ -64,12 +72,20 @@ var NetSimSlider = module.exports = function (rootDiv, options) {
   /**
    * A function invoked whenever the slider-value is changed by the student.
    * Passed the new value (not slider position) as an argument.
-   * @type function
+   * @type {function}
    * @private
    */
   this.changeCallback_ = utils.valueOr(options.onChange, function () {});
 
   /**
+   * A function invoked only when the slider-handle is released by the student.
+   * Passed the new value (not slider position) as an argument
+   * @type {function}
+   * @private
+   */
+  this.stopCallback_ = utils.valueOr(options.onStop, function () {});
```

Comment:
Hmm... that _is_ more descriptive.  Only reasons I'd push back are a) "stop" reflects jQueryUI's own naming convention, and b) the variable names several layers up the chain would get really, really cumbersome.